numpy/ma/core.py

```diff
@@ -5785,7 +5911,7 @@ def mini(self, axis=None):
             "`mini` is deprecated; use the `min` method or "
             "`np.ma.minimum.reduce instead.",
             DeprecationWarning, stacklevel=2)
-        return minimum.reduce(self, axis)
```

Comment:
Do we want to just remove `np.ma.mini()`? It has been deprecated since 1.13.0 it looks like.

Reply:
Yes please. If you are concerned about the size of the PR, it would be fine to do this in a separate PR. Note there is a commented out deprecated code in `numpy/ma/core.pyi` as well.